string name
float64[] prices
int64 auction_ID
